computer_vision_draft/houghTransform/houghTransform/main.cpp

```diff
@@ -0,0 +1,106 @@
+#include <opencv2/opencv.hpp>
+#include <iostream>
+#include <string>
+#include <fstream>
+#include<windows.h>  
+#include<chrono>
+#include<cmath>
+#include "DrivabilityDetector.h"
+
+using namespace cv;
+using namespace std;
+
+//Function for rotating 
```

Comment:
Try something like this

```cpp
/// Function for rotating. 
static Mat rotateImage(Mat image, double angle);
```

This should allow people who work on this in the future to simply hover over the function name to see what the function does.

The same goes for the other documentation comments. See this https://developer.lsst.io/cpp/api-docs.html for further information on doc comments

Reply:
rotateImage will not be used in final version. Short comment used.